docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_3.rst

```diff
@@ -23,6 +23,14 @@ Release notes: Open GEE 5.3.3
       removed for most calculations, such as when system manager is determining
       which resources overlap. 
 
+      **Reported memory used by the caches is more accurate**.

```

Comment:
Documentation is tricky. I'd suggest rewording this to make us sound less bad.  Maybe something like this:

```
**Improved memory usage reporting for caches**.
The memory usage estimation for the asset and
asset version caches has been improved and is
much more accurate.
```